source/game/kart/KartSub.cc

```diff
@@ -266,6 +275,26 @@ void KartSub::updateSuspOvertravel(const EGG::Vector3f &suspOvertravel) {
     m_minSuspOvertravel = m_minSuspOvertravel.maximize(suspOvertravel);
 }
 
+/// @addr{Inlined in 0x80596480}
```

Comment:
```suggestion
/// @addr{0x80598744}
```
Would it be appropriate to call this `tryEndHWG`?

Reply:
Well it handles both `m_bSoftWallDrift` and `m_bHWG` so `tryEndHWG` may not fully encompass the purpose of this function. I don't really have a strong grasp on what it means to have `m_bSoftWallDrift` enabled off-the-top of my head, so I'm not sure

Reply:
wait a second... I think I am not supposed to have the first half of the function body